vignettes/tcplfit2-vignette.Rmd

```diff
@@ -348,6 +348,17 @@ res_wide <- rbindlist(Map(cbind, spid = res$spid, res$V1))
 DT::datatable(res_wide,options = list(scrollX = T))
 ```
 
-*The same hitcalling can be done with the full data set, `dat`, as well.*
+*The same hitcalling can be done with the full data set, `dat`.*
 
+The output table resulting from the code in the last code chunk is the same format as the `res` table in example 3. Thus, users can use the `concRespPlot` function, as done previously in example 3, to plot the results. The next code chunk demonstrates how to visualize the example 4 fit results.
```

Comment:
Minor updates:
- [ ] "The output table resulting from the previous code chunk is ..."
- [ ] "Thus, one can use ..."